hs_core/templates/pages/metadata_terms.html

```diff
@@ -108,14 +108,28 @@
             <li><strong>Description: </strong> a text statement that points to a rights
                 management or usage statement for the resource.</li>
         </ul>
-
+        <a name="isCopiedFrom"></a>
         <ul>
             <li><strong>Term: </strong>URL</li>
             <li><strong>URI: </strong><a href="#URL">http://www.hydroshare.org/terms/URL</a></li>
             <li><strong>Label: </strong>Uniform Resource Locator</li>
             <li><strong>Description: </strong>URL that points to a rights
                 management or usage statement for the resource.</li>
         </ul>
+        <a name="isHostedBy"></a>
+        <ul >
+            <li><strong>Term: </strong>isCopiedFrom</li>
+            <li><strong>URI: </strong><a href="#isCopiedFrom">http://www.hydroshare.org/terms/isCopiedFrom</a></li>
+            <li><strong>Label: </strong>Copied From</li>
+            <li><strong>Description: </strong>The source URL or name where the resource is copied from.</li>
```

Comment:
The description text may be better as:

Name or URL of the resource from which a resource is copied.